### PR TITLE
docs build branch

### PR DESCRIPTION
**Pull request by abit2**
@Cadair

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -2,10 +2,32 @@
...
+IRISPy is a SunPy-affiliated package that provides the tools to read in and
+analyze data from the `IRIS <http://iris.lmsal.com>`__ solar-observing
+satellite in Python.
 
-.. automodapi:: irispy
-.. automodapi:: irispy.spectrograph
+The Interface Region Imaging Spectrograph is a NASA-funded Small Explorer which
+uses a high-frame-rate ultraviolet imaging spectrometer to make observations of
+the Sun. It provides 0.3 arcsec angular resolution and sub-angstrom spectral
+resolution at a one second cadence.
```
Comment (DanRyanIrish) [inline docs/index.rst]: Remove comment about one second cadence.  The cadence depends on the observing mode.  We don't have to go into the details of that here.

File: irispy/sji.py [hunk 2]
```diff
@@ -231,17 +232,17 @@ def __repr__(self):
...
-           date_end=self.date[-1], frame_num=len(self),
-           dim=u.Quantity(self.dimensions),
-           scale=u.Quantity(self.scale), obs_id=self.iris_obs_id, obs_desc=self.iris_obs_description,
-           tmf=TIME_FORMAT) + self.data.__repr__())
+                obs=self.observatory, inst=self.instrument, det=self.detector,
+                meas=self.measurement, wave=self.wavelength, date_start=self.date[0],
+                date_end=self.date[-1], frame_num=len(self),
+                dim=u.Quantity(self.dimensions),
+                scale=u.Quantity(self.scale), obs_id=self.iris_obs_id, obs_desc=self.iris_obs_description,
+                tmf=TIME_FORMAT) + self.data.__repr__())
```
Comment (DanRyanIrish) [inline irispy/sji.py]: Are these extra spaces meant to be here?  Are they a PEP8 issue?

File: irispy/spectrograph.py [hunk 3]
```diff
@@ -16,7 +16,27 @@
 
 
 class IRISSpectrograph(object):
-    """An object to hold data from multiple IRIS raster scans."""
+    """
+    A 2D IRIS Slit Jaw Imager Map.
```
Comment (DanRyanIrish) [inline irispy/spectrograph.py]: This is wrong.  IRISSPectrograph isn't a 2D Slit Jaw Map.  This should be replaces by L29.

File: docs/index.rst (hunk 1 above)
Comment (Cadair) [inline docs/index.rst]: I copied this out of the README ;)

File: irispy/sji.py (hunk 2 above)
Comment (abit2) [inline irispy/sji.py]: I think so, sublime text editor's  auto correction PEP8 made these changes.

Comment (DanRyanIrish) [inline irispy/sji.py]: OK then.

File: irispy/spectrograph.py (hunk 3 above)
Comment (abit2) [inline irispy/spectrograph.py]: Sorry I should have paid attention. I copied this from SJIMap :P

Comment (DanRyanIrish) [inline irispy/spectrograph.py]: No problem.  That's the point of code review.

**Comment by Cadair** [comment]
Can we merge this please? I wan't the IRISPy documentation online for the 0.8 release.

To be clear this (as a requirement to fix the docs) also moves the spectrograph object to using the new NDCube repo.

File: .travis.yml [hunk 4]
```diff
@@ -28,18 +28,21 @@ env:
...
-        - PIP_DEPENDENCIES='git+https://github.com/sunpy/cube.git'
+        - PIP_DEPENDENCIES='suds-jurko sunpy-sphinx-theme git+https://github.com/sunpy/ndcube.git'
         - EVENT_TYPE='pull_request push'
 
         # For this package-template, we include examples of Cython modules,
         # so Cython is required for testing. If your package does not include
         # Cython code, you can set CONDA_DEPENDENCIES=''
-        - CONDA_DEPENDENCIES='sunpy xarray pytest coverage sphinx sphinx_rtd_theme'
+        - SUNPY_DEPENDENCIES='openjpeg Cython jinja2 scipy matplotlib mock requests beautifulsoup4 sqlalchemy scikit-image pytest-mock pyyaml pandas nomkl pytest-cov coverage hypothesis glymur'
+        - CONDA_DEPENDENCIES='xarray pytest coverage sphinx sphinx_rtd_theme'
```
Comment (DanRyanIrish) [inline .travis.yml]: Remove xarray from here.  It's not a dependency anymore.

**Comment by DanRyanIrish** [comment]
@Cadair.  It looks fine to me.  Just remove xarray from the dependencies and we can merge.

**Pull request merged**